source/extensions/filters/network/postgres_proxy/postgres_decoder.cc

```diff
@@ -197,13 +217,10 @@ bool DecoderImpl::parseMessage(Buffer::Instance& data) {
     }
   }
 
-  setMessageLength(length);
-
   data.drain(startup_ ? 4 : 5); // Length plus optional 1st byte.
 
-  auto bytesToRead = length - 4;
+  auto bytesToRead = message_len_ - 4;
```

Comment:
existing state, but per style guide can you use an explicit type here rather than `auto`, and also spell this `bytes_to_read`.

Reply:
Agreed. I corrected all `auto` types to explicit ones when calculations are done to initialize the variable.